setup_support.py

```diff
@@ -100,12 +100,12 @@ def run(self):
             for cmdname in cmdline_command_names:
                 p = self._parser[cmdname]
                 cmdname = "{0}{1}".format(
-                    'datalad-' if cmdname != 'datalad' else '',
+                    'datalad ' if cmdname != 'datalad' else '',
                     cmdname)
                 format = cls(cmdname, ext_sections=sections, version=get_version())
                 formatted = format.format_man_page(p)
                 with open(opj(opath, '{0}.{1}'.format(
-                        cmdname,
+                        cmdname.replace(' ', '-'),
```

Comment:
the same as https://github.com/datalad/datalad-extension-template/blob/master/_datalad_buildsupport/setup.py#L154